src/lua/LuaPiGui.cpp

```diff
@@ -181,7 +181,7 @@ static LuaFlags<ImGuiInputTextFlags_> imguiInputTextFlagsTable = {
 	{ "AllowTabInput", ImGuiInputTextFlags_AllowTabInput },
 	{ "CtrlEnterForNewLine", ImGuiInputTextFlags_CtrlEnterForNewLine },
 	{ "NoHorizontalScroll", ImGuiInputTextFlags_NoHorizontalScroll },
-	{ "AlwaysInsertMode", ImGuiInputTextFlags_AlwaysInsertMode },
+	{ "AlwaysInsertMode", ImGuiInputTextFlags_AlwaysOverwrite },
```

Comment:
I'd appreciate it if you can Ctrl+F over the files in the data/ directory for usage of the AlwaysInsertMode name - if there are instances, please replace them with AlwaysOverwrite, otherwise please rename the string as well to match!

Reply:
There were no matches for "AlwaysInsertMode" in data/ so I just renamed the string in LuaPiGui.cpp :)

Reply:
Thanks! Just standard procedure when changing string script constants.